README.rst

```diff
@@ -0,0 +1,243 @@
+HiDimStat: High-dimensional statistical inference tool for Python
+=================================================================
+
+.. # Add the different badge
+
+|Linter&Tests| |CircleCI/Documentation| |CodeCov|
+
+|PyPi| |PyPi format| |PyPi_download| |Latest release| |PythonVersion|
+
+|GitHub contributors| |GitHub commit activity| |License| |Black|
+
+.. # Short description of the library
+
+The HiDimStat package provides statistical inference methods to solve the problem
+of variable importance evaluation in the context of predictive model using high-dimensional and spatially structured data.
+
+.. # Add usefull links
+
+If you like the package, spread the word and ⭐ our `official repository 
+<https://github.com/mind-inria/hidimstat>`_!
+
+Visit our website, https://hidimstat.github.io/, for more information.
+
+..
+  ## TODO: Add short citation when this will be ready
+  If you use HiDimStat for your published research, we kindly ask you to :ref:`cite<citation>` our article:
+  short reference
+
+Find your important variables in your data with the help of 
+`our examples <https://hidimstat.github.io/dev/auto_examples/index.html>`_.
+
+If you have any problems, please report them to the `GitHub issue tracker <https://github.com/mind-inria/hidimstat/issues>`_ 
+or contribute to the library by opening a pull request.
+
+Installation
+------------
+
+Dependencies
+^^^^^^^^^^^^
+
+.. # Add dependency of the project
+  TODO Need to match with pyproject.toml
+
+HiDimStat requires:
+
+- Python (>= |PythonMinVersion|)
+- joblib (>= |JoblibMinVersion|)
+- NumPy (>= |NumPyMinVersion|)
+- Pandas (>= |PandasMinVersion|)
+- Scikit-learn (>= |SklearnMinVersion|)
+- SciPy (>= |SciPyMinVersion|)
+
+HiDimStat's plotting capabilities require Matplotlib (>= |MatplotlibMinVersion|).
+
+To run the examples, Matplotlib (>= |MatplotlibMinVersion|) and seaborn (>=
+|SeabornMinVersion|) are required.
+
+User installation
+^^^^^^^^^^^^^^^^^
+
+.. # Add the instruction for installation
+  TODO add conda when it will be accessible
+
+HiDimStat can easily be installed via ``pip``. For more installation information,
+see the `installation instructions <https://hidimstat.github.io/dev/index.html#installation>`_.
+::
+  pip install -U hidimstat 
+
+Contribute
+----------
+
+.. # Add short discription for contribution to the library
+
+The best way to support the development of HiDimStat is to spread the word!
+
+HiDimStat aims to be supported by an active community, and we welcome 
+contributions to our code and documentation.
+
+For bug reports, feature requests, documentation improvements, or other issues, 
+you can create a `GitHub issue <https://github.com/mind-inria/hidimstat/issues>`_.
+
+If you want to contribute directly to the library, check the 
+`how to contribute <https://hidimstat.github.io/dev/how_to_contribute.html/>`_ page 
+on the website for more information.
+
+Quick start for development
```

Comment:
I think this section is also too specific to be in the readme. I suggest to create a dedicated page in the doc website or to put this in a separate readme.